MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/model/usermodel/Authorizations.java

```diff
@@ -0,0 +1,93 @@
+package com.mapbox.mapboxandroiddemo.model.usermodel;
+
+public class Authorizations {
```

Comment:
@langsmith 
Access can be package-private.
👀 the same thing happens in other classes inside `usermodel` package.